pkg/windows/windows.go

```diff
@@ -395,7 +396,11 @@ func (vm *windows) RebootAndReinitialize() error {
 	if _, err := vm.Run("Restart-Computer -Force", true); err != nil {
 		return fmt.Errorf("error rebooting the Windows VM: %w", err)
 	}
-	// Reinitialize the SSH connection after the VM reboot
+	// Wait for instance to be unreachable via SSH, implies reboot is underway
+	if err := vm.waitUntilUnreachable(); err != nil {
+		return fmt.Errorf("instance reboot failed to start: %w", err)
+	}
+	// Wait for instance to come back online and reinitialize the SSH connection after the reboot
```

Comment:
Is this comment accurate, where are we waiting for instance to be back?

Reply:
there's a [poll in init()](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/windows/connectivity.go#L87) retrying establishing an SSH connection, which is how we wait for the instance to be back online